web/templates/user/edit.html.haml

```diff
@@ -1,5 +1,5 @@
-.measure-wide.center
+.measure.center
   %h2.tc Edit user
   = render "form.html", changeset: @changeset, action: user_path(@conn, :update, @user)
```

Comment:
Should this be a partial? The filename isn't prefixed with an underscore `_`. Also is the .html part on this line necessary?